src/crypto/crypto_ec.cc

```diff
@@ -706,7 +706,42 @@ WebCryptoKeyExportStatus ECKeyExportTraits::DoExport(
     case kWebCryptoKeyFormatSPKI:
       if (key_data->GetKeyType() != kKeyTypePublic)
         return WebCryptoKeyExportStatus::INVALID_KEY_TYPE;
-      return PKEY_SPKI_Export(key_data.get(), out);
+      // Ensure exported key is in uncompressed point format.
+      // The temporary EC key is so we can have i2d_PUBKEY_bio() write out
+      // the header but it is a somewhat silly hoop to jump through because
+      // the header is for all practical purposes a static 26 byte sequence
+      // where only the second byte changes.
+      {
```

Comment:
Would it be possible to check for the key's point compression and only do this temporary key dance in a conditional block?

Reply:
Yes, but given that this probably isn't a performance critical code path, I lean towards simpler code with fewer branches.

Reply:
👍